app/views/api/v0/articles/index.json.jbuilder

```diff
@@ -1,19 +1,26 @@
 json.array! @articles do |article|
-  json.type_of            "article"
-  json.id                 article.id
-  json.title              article.title
-  json.description        article.description
-  json.cover_image        cloud_cover_url(article.main_image)
-  json.published_at       article.published_at
-  json.tag_list           article.cached_tag_list_array
-  json.slug               article.slug
-  json.path               article.path
-  json.url                article.url
-  json.canonical_url      article.processed_canonical_url
-  json.comments_count     article.comments_count
+  json.type_of                  "article"
+  json.id                       article.id
+  json.title                    article.title
+  json.description              article.description
+  json.cover_image              cloud_cover_url(article.main_image)
+  json.readable_publish_date    article.readable_publish_date
+  json.social_image             article_social_image_url(article)
+  json.tag_list                 article.cached_tag_list
+  json.tags                     article.cached_tag_list_array
```

Comment:
This can't be fixed right now, `tag_list` is not an array in the index, it's a comma separated list. I know it's different in the show but we can't break the interface with this PR. Could you keep this how it was?

It was mentioned in https://github.com/thepracticaldev/dev.to/issues/4206 and you can see the full discussion about it in the referred issue here https://github.com/thepracticaldev/dev.to/issues/2996

Reply:
Makes sense, thank you. I will revert it to `json.tag_list article.cached_tag_list_array`
Does it mean we shouldn't have `tags` fro index at all, only `tag_list`?

Reply:
I have removed `tags` for index :)

Reply:
Hi @nadiyaka, as long as the two are documented I think we can have both. It'd be awesome if you could add a test for the index, to make sure that no one accidentally or intentionally breaks the `tags` and `tag_list` fields by "fixing" the types

Thank you!

Reply:
Do you mean these for `show`:
```
json.tag_list                 @article.cached_tag_list
json.tags                     @article.cached_tag_list_array
```
and these for index:
```
json.tag_list                 article.cached_tag_list_array
json.tags                      article.cached_tag_list
```
?

Reply:
yeah, show stays like it is on master, index only adds `tags` as a `cached_tag_list`

thanks!